DirectEmployers/MyJobs
# [PD 3374] refactor analytics mobile css

## What's this PR do?
Refactors the mobile media queries for analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics apps

## How should this be manually tested?
Pull down repository. Go into analytics and make sure the css and look are the same

## Any background context you want to provide?
Need help from Aladeen as far as looking and seeing what I'm missing for fixing some of the CSS
